lib/matplotlib/tests/test_backend_macosx.py

```diff
@@ -16,11 +16,11 @@ def test_cached_renderer():
     # a fig.canvas.draw() call
     fig = plt.figure(1)
     fig.canvas.draw()
-    assert fig._cachedRenderer is not None
+    assert fig.canvas.get_renderer() is not None
```

Comment:
This is probably testing something slightly different now?

Reply:
Yeah, as the person who wrote this test originally I should have thought a bit more carefully about that ;) I think it is a bit of a moot point now with the reworked logic, but the real thing we want to make sure of is that when we call from the Objective-C code that we have the `_renderer()` buffer available, so I've stuck that here instead now.